document-generation/generate-document.ts

```diff
@@ -41,6 +43,8 @@ async function baseHandler(event: RequestEvent<GenerateDocumentRequest>): Promis
       return generateXlsx(event);
     case DocumentType.INCREMENTAL_FUNDING_PLAN:
       return generateWordDocument(event);
+    case DocumentType.EVALUATION_PLAN:
+      return generateWordDocument(event);
```

Comment:
I think these cases can be consolidate sense the same function is being called. And I started using `Docx`, so there might be a small merge conflict for the naming depending on when things are merged.

```typescript

    case DocumentType.INCREMENTAL_FUNDING_PLAN:
    case DocumentType.EVALUATION_PLAN:
      return generateWordDocument(event);
```

Reply:
Implemented in https://github.com/dod-ccpo/atat-web-api/pull/1381/commits/59e999694dcb21b3f0c4bca134d475dd4bb68221